pkg/controllers/kube-apiserver.go

```diff
@@ -225,19 +292,36 @@ func (s *KubeAPIServer) Run(ctx context.Context, ready chan<- struct{}, stopped
 		close(ready)
 	}()
 
-	// Work around that that the NewAPIServerCommand hardcodes the stop channel to SIGTERM signals,
-	// so we cannot use the cobra RunE command directly.
-	completedOptions, err := kubeapiserver.Complete(s.serverOptions)
+	fd, err := os.CreateTemp("", "kube-apiserver-config-*.yaml")
```

Comment:
What if instead of writing the config to a temp file, we store it under `/var/lib/microshift/` so users could modify it? The main problem (or advantage...) is that everytime MicroShift is restarted, the config file will be written as it's in the code base. We don't check if file is created, skip creation.

Reply:
We don't want users to modify (or even know about!) this file. In OpenShift, it's generated dynamically by the target config controller in cluster-kube-apiserver-operator (https://github.com/openshift/cluster-kube-apiserver-operator/blob/65fc794b8aaa36a881323b26618ab46b7ed7defd/pkg/operator/targetconfigcontroller/targetconfigcontroller.go) based on observed configuration.